cimbar/encode/cimb_translator.py

```diff
@@ -22,10 +23,10 @@ def possible_colors(dark, bits=0):
         ]
     elif dark and bits < 3:
         colors = [
+            (0, 0xFF, 0),
```

Comment:
I'll need to add back the old color set in a follow up PR. I'm thinking of merging the `dark` flag into something like a `color_mode` flag. So we'd have 0 for the original color arrangement, 1 for this, and 100 (or something) for light mode.